scriptmodules/emulators/scummvm-sdl1/01_rpi_sdl1.diff

```diff
@@ -1,11 +1,13 @@
---- a/configure	2018-08-22 14:35:59.974063916 +0000
-+++ b/configure	2018-08-22 14:36:36.379862346 +0000
-@@ -3274,7 +3274,7 @@
+diff --git a/configure b/configure
+index 6f44008abdb..25b3713e243 100755
+--- a/configure
++++ b/configure
+@@ -3379,7 +3379,7 @@ if test -n "$_host"; then
  			# We prefer SDL2 on the Raspberry Pi: acceleration now depends on it
  			# since SDL2 manages dispmanx/GLES2 very well internally.
  			# SDL1 is bit-rotten on this platform.
 -			_sdlconfig=sdl2-config
 +			_sdlconfig=sdl-config
```

Comment:
BTW: I was wondering whether it would be better to achieve the result of the patch with a simple one-line sed command instead. In my opinion, this would be easier to maintain since such a sed scriptlet is more robust against small changes in the context of the change but for sure is also more likely to get silently ignored if the structure of the code gets changed in a substantial way. What is your opinion on that?

Reply:
To be more precise I thought about replacing the patch with something like this:
sed -ie 's/\([[:space:]]_sdlconfig=sdl\)2\(-config\)/\1\2/' configure

Reply:
`sed` would have worked, I think even with a simpler substitution.
However, I think the better option would be to pre-set `SDL_CONFIG` when calling `configure`, since [find_sdl](https://github.com/scummvm/scummvm/blob/4da739e26e43de2e7fd2efbf4c026c6b14493c09/configure#L451-L454) will prioritize it before `$_sdlconfig`.
I.e.
```
SDL_CONFIG=sdl-config ./configure --params
```
will pick `sdl1` before `sdl2`. This will allow use to drop the patches for `configure.